roles/installer/defaults/main.yml

```
@@ -401,7 +401,7 @@ postgres_init_container_resource_requirements:
     memory: 64Mi
 # Assign a preexisting priority class to the postgres pod
 postgres_priority_class: ''
-postgres_data_path: '/var/lib/postgresql/data/pgdata'
+postgres_data_path: '/var/lib/pgsql/data/pgdata'
```

This will conflicts with https://github.com/ansible/awx-operator/pull/1798, and the path is incorrect.

```suggestion
postgres_data_path: '/var/lib/pgsql/data/userdata'
```